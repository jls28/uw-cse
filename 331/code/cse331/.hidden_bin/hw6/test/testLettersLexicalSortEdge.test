# Should go through edge 'vowel' instead of edge 'xoxo'

LoadGraph letters letters.tsv
FindPath letters a g