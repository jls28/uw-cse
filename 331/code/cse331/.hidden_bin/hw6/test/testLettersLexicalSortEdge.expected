# Should go through edge 'vowel' instead of edge 'xoxo'

loaded graph letters
path from a to g:
a to y via vowel
y to g via consonant